google-fcm/src/main/scala/akka/stream/alpakka/google/firebase/fcm/javadsl/GoogleFcmFlow.scala

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (C) 2016-2018 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.google.firebase.fcm.javadsl
+
+import akka.NotUsed
+import akka.actor.ActorSystem
+import akka.http.scaladsl.Http
+import akka.japi.Pair
+import akka.stream.alpakka.google.firebase.fcm.FcmFlowModels.{FcmFlowConfig, FcmResponse}
+import akka.stream.alpakka.google.firebase.fcm.impl.FcmSender
+import akka.stream.alpakka.google.firebase.fcm.{FcmFlowModels, FcmNotification}
+import akka.stream.scaladsl.{Flow, Sink}
+import akka.stream.{javadsl, Materializer}
+
+object GoogleFcmFlow {
+  def sendWithData[T](
```

Comment:
In other connectors this is called `...WithPassThrough`. (And the typical use case is to commit to Kafka after data is pushed somewhere else.)

Reply:
In this case, when you fail a send maybe you want to resend/reschedule. And the returning error message tells you nearly nothing about what was the input data. This was the main reason I created this function. (And I read it in the "good practices" section too ;) )